trieste/acquisition/function.py

```diff
@@ -32,26 +31,10 @@
 """
 Type alias for acquisition functions.
 
-An `AcquisitionFunction` maps a single query point (of dimension `D`) to a single value that
-describes how useful it would be evaluate that point (to our goal of optimizing the objective
-function). Thus, with leading dimensions, an `AcquisitionFunction` takes input shape
-`[..., D]` and returns shape `[..., 1]`.
-
-**Note:** Type checkers will not be able to distinguish an `AcquisitionFunction` from a
-`BatchAcquisitionFunction`.
-"""
-
-BatchAcquisitionFunction = Callable[[TensorType], TensorType]
-"""
-Type alias for batch acquisition functions.
-
-A `BatchAcquisitionFunction` maps a set of `B` query points (each of dimension `D`) to a single
+An :const:`AcquisitionFunction` maps a set of `B` query points (each of dimension `D`) to a single
 value that describes how useful it would be evaluate all these points together (to our goal of
-optimizing the objective function). Thus, with leading dimensions, a `BatchAcquisitionFunction`
+optimizing the objective function). Thus, with leading dimensions, an :const:`AcquisitionFunction`
 takes input shape `[..., B, D]` and returns shape `[..., 1]`.
```

Comment:
I'd like to add something upfront to say that non-batch acq fct should be defined with batch 1

Reply:
sounds sensible